File: ads-section-3/programação-orientada-a-objeto/c++/programa.cpp
#include <iostream>

int main(int argc, char **argv) {
	int qtdePedido;
	int qtdeEstoque;
	int qtdeMinimaPorVenda;

	bool vendaLiberada = qtdePedido <= qtdeEstoque && qtdePedido >= qtdeMinimaPorVenda;

	std::cout << "Entre com o valor do pedido: ";
	std::cin >> qtdePedido;
	
	std::cout << "Entre com o valor do estoque: ";
	std::cin >> qtdeEstoque;

	std::cout << "Entre com o valor minimo por venda: ";
	std::cin >> qtdeMinimaPorVenda;

	std::cout << "Pedido lierado: " << (vendaLiberada ? "True" : "False") << std::endl;

}

File: ads-section-3/programação-orientada-a-objeto/java/Programa.java
import java.util.Scanner;

public class Programa {
	public static void main(String[] args) {
		Scanner leitor = new Scanner(System.in);

		int qtddePedido;
		int qtddeEstoque;
		int qtdeMinimaPorVenda;

		System.out.print("Entre com o valor do pedido: ");
		qtddePedido = leitor.nextInt();

		System.out.print("Entre com o valor do estoque: "); 
		qtddeEstoque = leitor.nextInt(); 

		System.out.print("Entre com a quantidade mínima por venda: ");
		qtdeMinimaPorVenda = leitor.nextInt();

		leitor.close();
		boolean vendaLiberada = qtddePedido <= qtddeEstoque && qtddePedido >= qtdeMinimaPorVenda;

		System.out.println("Pedidod Liberado: "+vendaLiberada);
	}
}
